.github/workflows/code_metrics_pr.yml

```diff
@@ -6,6 +6,10 @@ jobs:
   code-metrics:
     runs-on: ubuntu-latest
```

Comment:
Just a comment, do not need to fix the code: I think the base image `ubuntu-latest` would be a bit heavy to handle the following simple tasks. If we have a chance let's improve it in the future.

Reply:
@kmiya What do you think can be concretely used as an alternative?

Reply:
I thought `python3.x-alpine` or something would be more appropriate for this kind of simple tasks, but just have noticed that the (remote) size of `ubuntu-latest` is only [28.6 MB](https://github.com/docker-library/repo-info/blob/master/repos/ubuntu/remote/latest.md). It was much smaller than I thought, sorry, please ignore the above comment!